libamqpprox/amqpprox_maybesecuresocketadaptor.h

```diff
@@ -60,14 +63,16 @@ class MaybeSecureSocketAdaptor {
     }
```

Comment:
This constructor is only for testing. But I guess, still we should initialise `d_smallBuffer` and `d_smallBufferSet` with default values.